ACE/include/makeinclude/platform_macosx.GNU

```diff
@@ -42,10 +44,15 @@ else ifeq ($(MACOS_MAJOR_VERSION),11)
     ## just use the latest known version
     MACOS_CODENAME = $(MACOS_CODENAME_VER_11_latest)
   endif
+else ifeq ($(MACOS_MAJOR_VERSION),12)
+  ifeq ($(shell test $(MACOS_MINOR_VERSION) -gt 2; echo $$?),0)
+    ## if the detected version is greater than the latest know version,
+    ## just use the latest known version
+    MACOS_CODENAME = $(MACOS_CODENAME_VER_12_latest)
+  endif
 else
-  ## if the detected version is greater than the latest know version,
-  ## just use the latest known version
-  MACOS_CODENAME = $(MACOS_CODENAME_VER_11_latest)
+  ## Unsupported major version
```

Comment:
Let us fallback to 12_latest when we have an unknown macos major version (for example 13)

Reply:
Done!